app/workers/evss/disability_compensation_form/submit_form0781.rb

```diff
@@ -42,38 +57,41 @@ class SubmitForm0781 < Job
       # `file`, which is the generated file location
       def get_docs(submission_id, uuid)
         @submission_id = submission_id
-        parsed_forms = JSON.parse(submission.form_to_json(Form526Submission::FORM_0781))
-        file_type_and_file_objs = []
+        @uuid = uuid
```

Comment:
making this an instance variable makes the value available to the instance level logging.